conf/modules.config

```diff
@@ -534,14 +534,28 @@ process {
     }
 
     withName: MOTUS_MERGE {
-        ext.args = { params.generate_biom_output ? "-B" : "" }
+        ext.args = { params.standardisation_motus_generatebiom ? "-B" : "" }
         ext.prefix = { "motus_${meta.id}_combined_reports" }
         publishDir = [
             path: { "${params.outdir}/motus/" },
             mode: params.publish_dir_mode
         ]
     }
 
+    withName: TAXPASTA_MERGE {
+        ext.args =  {
+            [
+                "-p ${meta.tool} -o ${meta.tool}_${meta.id}.${params.standardisation_taxpasta_format}",
+                params.standardisation_taxpasta_addtaxonomy ? "-p" : ""
```

Comment:
I guess this is not valid currently? Also it would be incorrect as `-p` is for profile?

Reply:
I will remove it for this release and yes the argument is `--taxonomy`

Reply:
Or shall I keep it?

Reply:
We should probably remove it everywhere, I don't thin ktaxonomy support will make it for pipeline release

Reply:
Then I will remove it for the time being.